mongodbatlas/organizations_test.go

```diff
@@ -0,0 +1,354 @@
+// Copyright 2019 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mongodbatlas
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/go-test/deep"
+)
+
+func TestOrganizationsServiceOp_List(t *testing.T) {
+	client, mux, teardown := setup()
+	defer teardown()
+
+	mux.HandleFunc("/orgs", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, http.MethodGet)
+		_, _ = fmt.Fprint(w, `{
+			"links": [{
+				"href": "https://cloud.mongodb.com/api/public/v1.0/orgs",
+				"rel": "self"
+			}],
+			"results": [{
+				"id": "56a10a80e4b0fd3b9a9bb0c2",
+				"links": [{
+					"href": "https://cloud.mongodb.com/api/public/v1.0/orgs/56a10a80e4b0fd3b9a9bb0c2",
+					"rel": "self"
+				}],
+				"name": "012i3091203jioawjioej"
+			}, {
+				"id": "56aa691ce4b0a0e8c4be51f7",
+				"links": [{
+					"href": "https://cloud.mongodb.com/api/public/v1.0/orgs/56aa691ce4b0a0e8c4be51f7",
+					"rel": "self"
+				}],
+				"name": "1454008603036"
+			}],
+			"totalCount": 2
+		}`)
+	})
+
+	orgs, _, err := client.Organizations.List(ctx, nil)
+	if err != nil {
+		t.Fatalf("Organizations.List returned error: %v", err)
+	}
+
+	expected := &Organizations{
+		Links: []*Link{
+			{
+				Href: "https://cloud.mongodb.com/api/public/v1.0/orgs",
+				Rel:  "self",
+			},
+		},
+		Results: []*Organization{
+			{
+				ID: "56a10a80e4b0fd3b9a9bb0c2",
+				Links: []*Link{
+					{
+						Href: "https://cloud.mongodb.com/api/public/v1.0/orgs/56a10a80e4b0fd3b9a9bb0c2",
+						Rel:  "self",
+					},
+				},
+				Name: "012i3091203jioawjioej",
+			},
+			{
+				ID: "56aa691ce4b0a0e8c4be51f7",
+				Links: []*Link{
+					{
+						Href: "https://cloud.mongodb.com/api/public/v1.0/orgs/56aa691ce4b0a0e8c4be51f7",
+						Rel:  "self",
+					},
+				},
+				Name: "1454008603036",
+			},
+		},
+		TotalCount: 2,
+	}
+
+	if diff := deep.Equal(orgs, expected); diff != nil {
+		t.Error(diff)
+	}
+}
+
+func TestOrganizationsServiceOp_Get(t *testing.T) {
+	client, mux, teardown := setup()
+	defer teardown()
+
+	ID := "5a0a1e7e0f2912c554080adc"
+
+	mux.HandleFunc(fmt.Sprintf("/%s/%s", orgsBasePath, ID), func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, http.MethodGet)
+		_, _ = fmt.Fprint(w, `{
+		"id": "56a10a80e4b0fd3b9a9bb0c2",
```

Comment:
[np] doesn't change the results of the test but this should match ID above right?